manifests/config.pp

```diff
@@ -25,20 +25,9 @@
     content => template('foreman/database.yml.erb'),
   }
 
-  if $::foreman::email_delivery_method and !empty($::foreman::email_delivery_method) {
-    if $::foreman::email_config_method == 'file' {
-      file { "/etc/foreman/${foreman::email_conf}":
-        ensure  => file,
-        owner   => 'root',
-        group   => $::foreman::group,
-        mode    => '0640',
-        content => template("foreman/${foreman::email_source}"),
```

Comment:
You can also remove the template itself I believe.

Reply:
Done